etl/afd_ems_import/utils/columns.py

```diff
@@ -0,0 +1,80 @@
+COLUMNS = {
+    "ems": {
+        "cols_to_rename": {
+            "mvc_form_extrication_time": "mvc_form_extrication_datetime"
+        },
+        "update_columns": [
```

Comment:
i've been thinking a lot about these records and how we need to consider personal privacy as we look at linking EMS incidents and CR3s. there are a handful of columns here that potentially contain personal health information and are of questionable utility to us. i've started a conversation with Xavier about removing as many columns as possible from this table. that will be addressed in a separate PR.